build.cake

```diff
@@ -75,7 +75,7 @@ Task("Test")
     OpenCover(
         tool => tool.DotNetCoreVSTest("**/bin/**/*Tests.dll"),
         new FilePath("./coverage.xml"),
-        new OpenCoverSettings { ReturnTargetCodeOffset = 0, OldStyle = true }.WithFilter("+[*LevelUp*]*").WithFilter("-[*Tests*]*")
+        new OpenCoverSettings { ReturnTargetCodeOffset = 0, OldStyle = true }.WithFilter("+[*LevelUp*]LevelUp*").WithFilter("-[*Tests*]*")
```

Comment:
I guess I don't understand how the filter works, but `.WithFilter("+[*LevelUp*]LevelUp*").WithFilter("-[*Tests*]*")` - the `LevelUp*` seems like it would be superfluous...

Reply:
Inside the `[]` is an assembly name match, outside the `[]` is a classname match.